src/processing/code_parser.py

```diff
@@ -0,0 +1,9 @@
+from abc import ABC
+from typing import Dict, Any
+
+from processing.data_processor import DataProcessor
+
+
+class CodeParser(DataProcessor, ABC):
+    def extract_from_raw_data(self, raw_json_object: Dict[str, Any]) -> Any:
+        return raw_json_object['code']
```

Comment:
We'll need to make sure we skip the docstring node, since the authors' model predicts from 'code_tokens' which does skip the docstring node. But it is not a concern of this parser class.

Reply:
We can filter docstrings later based on the node types.